### PR TITLE
Cleanup references to deprecated zones

### PR DESCRIPTION
**Pull request by pierresegonne**
Follow-up on https://github.com/electricitymap/electricitymap-contrib/pull/3817 
All references to deleted zones had not been removed

### CODE REVIEW

**Review by madsnedergaard — APPROVED** [review]
💪

File: config/co2eq_parameters_direct.json [hunk 1]
```diff
@@ -3187,9 +3187,9 @@
           }
         ],
         "unknown": {
-            "_url": "https://docs.google.com/spreadsheets/d/1CegROfej9HqRZTfihpjPpgZTYPUHrTNgTdHWbbP3w74/edit#gid=291258352",
```
Comment (pierresegonne) [inline config/co2eq_parameters_direct.json:3190]: Could we also run formatting on json files?

Comment (VIKTORVAV99) [reply config/co2eq_parameters_direct.json:3190]: We can run prettier on them. I was planning on adding that and a action for it after #4171 is merged.

Comment (pierresegonne) [reply config/co2eq_parameters_direct.json:3190]: That would be awesome yeah!

**Pull request merged**